server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -1468,5 +1468,9 @@ func (s *BuildBuddyServer) GetAuditLogs(ctx context.Context, request *alpb.GetAu
 }
 
 func (s *BuildBuddyServer) CreateRepo(ctx context.Context, request *repb.CreateRepoRequest) (*repb.CreateRepoResponse, error) {
-	return nil, status.UnimplementedError("Not implemented")
+	gh := s.env.GetGitHubApp()
+	if gh == nil {
+		return nil, status.FailedPreconditionError("github app not enabled in the server")
+	}
```

Comment:
if you wanted to be consistent with other handlers you could do:

```
	if gh == nil {
		return nil, status.UnimplementedError("Github service not configured")
	}
```

Reply:
done, i just copied the ones above me - but i went ahead and updated those too